app/src/main/java/nl/tudelft/b_b_w/model/HashException.java

```diff
@@ -3,5 +3,5 @@
 /**
  * Thrown when the hash methods are not available.
  */
-public class HashException extends Exception {
+public class HashException extends RuntimeException {
```

Comment:
In this case it is because the hash exception is only thrown when either UTF-8 is not available or when SHA-256 is not available. In both of these cases the hash function will not work making our app completely useless. When we throw this exception, we have to change every signature to catch this rare exception, and when we finally catch it, the best we can do is "Sorry, the hashing function is not available (in your country)" and then close the app. Therefore I think in this case a RuntimeException is acceptable.